webapp/src/components/AuctionPage/AuctionPage.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Message,
+  Header,
+  Grid,
+  Container
+  // Button
```

Comment:
I could! But I left it there because on of the other PRs up ahead uses it and I didn't want to rebase everything.

But I'll have to rebase anyways so I'll do it